### PR TITLE
4.x remove deprecation code

### PR DESCRIPTION
**Pull request by saeideng**
removing `ignoreMissing` as it now is useless

### CODE REVIEW

**Comment by saeideng** [comment]
I forgot to remove `ignoreMissing` from tests 
I'll do

**Review by markstory — COMMENTED** [review]
Looking good.

**Comment by saeideng** [comment]
I removed unused `tests/test_app/TestApp/Template/Scaffold/empty` file

File: src/View/Helper/PaginatorHelper.php [hunk 1]
```diff
@@ -495,20 +495,11 @@ public function sort($key, $title = null, array $options = [])
      * @param array $options Pagination/URL options array
      * @param string|null $model Which model to paginate on
      * @param array $urlOptions Array of options
-     * The bool version of this argument is *deprecated* and will be removed in 4.0.0
      * @return string By default, returns a full pagination URL string for use in non-standard contexts (i.e. JavaScript)
      * @link https://book.cakephp.org/3.0/en/views/helpers/paginator.html#generating-pagination-urls
      */
     public function generateUrl(array $options = [], $model = null, $urlOptions = [])
```
Comment (dereuromark) [inline src/View/Helper/PaginatorHelper.php]: typehint array should be added for urlOptions.

**Pull request merged**